components/ActionBox/components/DealActionBox.tsx

```diff
@@ -0,0 +1,265 @@
+		) {
+			max =
+				Number(max) >=
+				dealRedemptionData?.purchaseTokenTotalForDeal - dealRedemptionData?.totalAmountAccepted
+					? dealRedemptionData?.purchaseTokenTotalForDeal - dealRedemptionData?.totalAmountAccepted
+					: 0;
+		}
+
+		if (
+			(dealRedemptionData?.status === Status.Closed ||
+				(dealRedemptionData?.status === Status.OpenRedemption &&
+					!dealRedemptionData.isOpenEligible)) &&
+			!isWithdraw
+		) {
+			max = 0;
+		}
+
+		setIsMaxValue(true);
+		setInputValue(Number(max));
+	};
+
+	return (
+		<Container>
+			<RedemptionHeader>
+				<RedemptionPeriodTooltip
+					visible={showTooltip}
+					appendTo="parent"
+					trigger="click"
+					allowHTML
+					interactive
+					content={
+						<div>
+							A max pro rata contribution makes you eligible for the open redemption period where
+							unredeemed deal tokens are available
+						</div>
+					}
+				>
+					<FlexDivRowCentered>
+						{statusToText(dealRedemptionData.status)}
+						<InfoClick onClick={() => setShowTooltip(!showTooltip)}>
+							<Image src={Info} alt="info icon" />
+						</InfoClick>
+					</FlexDivRowCentered>
+				</RedemptionPeriodTooltip>
+			</RedemptionHeader>
+			<ContentContainer>
+				<>
+					<ActionBoxInputLabel>{label}</ActionBoxInputLabel>
+					<InputContainer>
+						<ActionBoxInput
+							type={'number'}
+							placeholder={placeholder}
+							value={inputValue}
+							onChange={(e) => {
+								const value = !!e.target.value.length ? parseFloat(e.target.value) : null;
+								setIsMaxValue(false);
+								setInputValue(value);
+							}}
+						/>
+						{maxValue && (
+							<ActionBoxMax
+								isProRata={dealRedemptionData?.status === Status.ProRataRedemption && !isWithdraw}
+								onClick={handleMaxButtonClick}
+							>
+								{dealRedemptionData?.status === Status.ProRataRedemption && !isWithdraw
+									? 'Max Pro Rata'
+									: 'Max'}
+							</ActionBoxMax>
+						)}
+					</InputContainer>
+					<ActionBoxHeaderWrapper>
+						<ActionBoxHeader
+							isAcceptOrReject
+							isPool={false}
+							isSelected={isDealAccept}
+							onClick={() => setIsDealAccept(true)}
+						>
+							<FlexDivCenterRow>
+								<>Accept Deal</>
+								<QuestionMark
+									text={`Choose accept to agree to the deal terms with up to the max amount based on your allocation this round`}
+								/>
+							</FlexDivCenterRow>
+						</ActionBoxHeader>
+						<ActionBoxHeader
+							isAcceptOrReject
+							isPool={false}
+							isSelected={!isDealAccept}
+							isWithdraw={isWithdraw}
+							onClick={() => setIsDealAccept(false)}
+						>
+							<FlexDivCenterRow>
+								<>Withdraw</>
+								<QuestionMark text="Withdraw a portion or all of your capital from the pool and receive your original purchase tokens back" />
+							</FlexDivCenterRow>
+						</ActionBoxHeader>
+					</ActionBoxHeaderWrapper>
+				</>
+			</ContentContainer>
+			{poolId !== '0x7e135d4674406ca8f00f632be5c7a570060c0a15' && (
```

Comment:
Surely we can put this pool ID in a constant file somewhere and explain why it is here

Reply:
Removed